### ISSUE TITLE
How can I get predict values in train set

### ISSUE THREAD

**Issue by JaneTakanashi** [guoguibing/librec#256]
All the results shows in is the test set, and how can I get predict values in the train set

**Comment by allenjack** [comment]
Hi,

You can set your training set also be the testing set.

**Comment by JaneTakanashi** [comment]
you mean in the testset mode, I put the train set and test set both in train dir and test dir?

**Comment by allenjack** [comment]
Yes, exactly. It should work.
If you have any further questions, please let us know.
